hpat/tests/test_series.py

```diff
@@ -2099,6 +2099,48 @@ def test_series_nunique_param1_impl(S, dropna):
                     result_param1 = hpat_func_param1(S, param1)
                     self.assertEqual(result_param1, result_param1_ref)
 
+    def test_series_count(self):
```

Comment:
The test executes  [the old functionality](https://github.com/IntelPython/hpat/blob/master/hpat/hiframes/pd_series_ext.py#L504-L506). You should comment out that to be able to to test the implementation in new style.